fvtest/tril/tril/ilgen.cpp

```diff
@@ -184,11 +185,34 @@ TR::Node* Tril::TRLangBuilder::toTRNode(const ASTNode* const tree) {
         }
      }
      else if (opcode.isLoadIndirect() || opcode.isStoreIndirect()) {
-         auto offset = tree->getArgByName("offset")->getValue()->get<int32_t>();
+         // If not specified, offset will default to zero. 
+         int32_t offset = 0; 
+         if (tree->getArgByName("offset")) {
+            offset = tree->getArgByName("offset")->getValue()->get<int32_t>();
+         } else { 
+            // warning("Should specify offset on indirect load and stores -- has defaulted to zero"); 
+         }
+
          TraceIL("  is indirect store/load with offset %d\n", offset);
          const auto name = tree->getName();
-         auto type = opcode.getType();
          auto compilation = TR::comp();
+         TR::DataType type;  
+         if (opcode.isVector()) { 
```

Comment:
Please add a trace message here to signal that we've detected a vector opcode.

Reply:
See earlier reply about trace.